changelog.d/775.misc

```diff
@@ -0,0 +1 @@
+Require node 20, update dependencies.
```

Comment:
This needs to be a `.removal`, as we're removing support. I'd put in stronger wording to say that Node 16 and 18 are no longer supported, and users need to update to Node 20.

I wouldn't both stating that dependencies are updated, it doesn't help the user any to know that.